README.md

```diff
@@ -219,6 +220,20 @@ await client.logon();
 await client.logoff();
 ```
 
+## refreshClient callback
+The refreshClient is an async callback function with the sdk client as parameter, it is called when the ACC session is expired.
+The callback must refresh te client session and return it. if a SOAP query fails with session expiration error then it will be retried when the callback is defined. 
+
+```js
+const connectionParameters = sdk.ConnectionParameters.ofUserAndPassword(
+                                url, "admin", "admin",
+                                {refreshClient: async (client)=>{
```

Comment:
Please respect indentation and spacing to be consistent with the rest of the code.

Reply:
done